hack/tests/e2e-helm.sh

```diff
@@ -6,92 +6,84 @@ source hack/lib/test_lib.sh
 source hack/lib/image_lib.sh
 
 DEST_IMAGE="quay.io/example/nginx-operator:v0.0.2"
-ROOTDIR="$(pwd)"
 TMPDIR="$(mktemp -d)"
 trap_add 'rm -rf $TMPDIR' EXIT
 
-test_namespace="test-e2e-helm"
+# kind has an issue with certain image registries (ex. redhat's), so use a
+# different test pod image.
+METRICS_TEST_IMAGE="fedora:latest"
 
-deploy_operator() {
-    kubectl create -f "$OPERATORDIR/deploy/crds/helm.example.com_nginxes_crd.yaml"
-    kubectl create -f "$OPERATORDIR/deploy/service_account.yaml"
-    kubectl create -f "$OPERATORDIR/deploy/cluster_role.yaml"
-    kubectl create -f "$OPERATORDIR/deploy/cluster_role_binding.yaml"
-    kubectl create -f "$OPERATORDIR/deploy/cluster_operator.yaml"
-    kubectl create namespace ${test_namespace}
-}
+# Namespace used to deploy the helm project
+test_namespace="nginx-operator-system"
```

Comment:
`test_namespace` is the namespace in which to create the CR, not the operator. The e2e test ensures that the operator properly handles managing CRs in other namespaces. So this needs to be different than the operator namespace.

Reply:
since `make deploy` creates and deploys into `nginx-operator-system`, we need a separate variable (e.g. `operator-namespace`) for this value.

We should use `test_namespace` everywhere we were using it before, and we should use `operator_namespace` whenever we made a kubectl call without an explicit `--namespace` flag before.

Reply:
Done 👍